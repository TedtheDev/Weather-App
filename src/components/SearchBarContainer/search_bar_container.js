import React, { Component } from 'react';

import SearchBar from './SearchBar/search_bar';

class SearchBarContainer extends Component {
    constructor(props) {
        super(props);
        
    }

    render() {
        return (
            <SearchBar 
                getForecast={this.props.getForecast} 
                onChangeCityName={this.props.onChangeCityName}
                cityName={this.props.cityName}
                error={this.props.error}
            />
        )
    }
}

export default SearchBarContainer;