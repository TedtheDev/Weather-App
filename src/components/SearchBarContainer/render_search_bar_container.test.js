import React from 'react';
import SearchBarContainer from './search_bar_container';
import renderer from 'react-test-renderer';

describe('<SearchBarContainer />', () => {
    test('renders correctly', () => {
        const component = renderer.create(<SearchBarContainer getForecast={jest.fn} onChangeCityName={jest.fn}/>);

        const tree = component.toJSON();

        expect(tree).toMatchSnapshot();
    });
});