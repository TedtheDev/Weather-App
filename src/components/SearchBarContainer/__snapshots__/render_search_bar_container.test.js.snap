// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SearchBarContainer /> renders correctly 1`] = `
<form
  className="sc-bdVaJa fAOpag"
  onSubmit={[Function]}
>
  <input
    className="sc-bwzfXH khaJxb"
    onChange={[Function]}
    placeholder="City Name"
    value={undefined}
  />
  <div
    className="sc-bxivhb cfEzOD"
  />
  <button
    className="sc-htpNat dkRZgH"
    type="submit"
  >
    Get Forecast
  </button>
</form>
`;
